daemon/src/db.rs

```diff
@@ -637,7 +453,7 @@ mod tests {
             .unwrap();
 
         let cfds_from_db = load_all_cfds(&mut conn).await.unwrap();
-        let cfd_from_db = cfds_from_db.first().unwrap().clone();
+        let cfd_from_db = cfds_from_db.last().unwrap().clone();
```

Comment:
Why did we need to change this test?

This looks like a regression!

Reply:
This was changed because the the SQL had an unnecessary reverse-ordering step:
```
        where cfd_states.state in (
            select
              state
              from cfd_states
            where cfd_id = cfds.id
            order by id desc
            limit 1
        )
```

With the reversal step removed, the test item moves from `.first()` to `.last()` is all.